orga/tests/unit/models/campaign-test.js

```diff
@@ -86,43 +86,29 @@ module('Unit | Model | campaign', function(hooks) {
     });
   });
 
-  module('canBeArchived', function() {
+  module('isArchived', function() {
     let store;
 
     hooks.beforeEach(function() {
       store = this.owner.lookup('service:store');
     });
 
-    module('when type is ASSESSMENT', function() {
-      module('when campaign is not archived', function() {
-        test('it should return true', function(assert) {
-          const campaign = store.createRecord('campaign', {
-            type: 'ASSESSMENT',
-            archivedAt: null,
-          });
-
-          assert.equal(campaign.canBeArchived, true);
-        });
-        module('when campaign is archived', function() {
-          test('it should return false', function(assert) {
-            const campaign = store.createRecord('campaign', {
-              type: 'ASSESSMENT',
-              archivedAt: new Date('2020-01-01'),
-            });
-
-            assert.equal(campaign.canBeArchived, false);
-          });
+    module('when campaign is not archived', function() {
```

Comment:
si je lis le module et le test
ça donne  :
> when campaign is not archived it should return false
when campaign is archived it should return true

je trouverais mieux d'avoir 

> when campaign has no archived date it should return false
when campaign has an archived date it should return true

Ou 


> when campaign has no archived date campaign is not archived
when campaign has an archived date campaign is archived